managed/ui/src/routes.js

```diff
@@ -63,7 +65,9 @@ export const clearCredentials = () => {
   Cookies.remove('authToken');
   Cookies.remove('customerId');
   Cookies.remove('userId');
-  browserHistory.push('/');
+  pathToRedirect
```

Comment:
clearCredential is not right place to clear queryParam, minor comment. If you can please either rename the function or pull this snippet in another function.

Reply:
I am not clearing the `queryParams` in `clearCredential` function, but it's a check that we should load the page with params intact if we have the query param set previously. Actual clear logic is in `validateSession` function once it redirects.

Reply:
The reason for this check is because the login component is calling clear credential and thus this will preserve the `queryParam`

Reply:
Cool